src/Krank/Checkers/IssueTracker.hs

```diff
@@ -148,6 +152,7 @@ statusParser (AesonT.Object o) = do
       readState (AesonT.Success status) = case status of
         "closed" -> Right Closed
         "open"   -> Right Open
+        "opened" -> Right Open
```

Comment:
Please add a comment here saying here that GitHub have "closed" and "open" tokens. And that gitlzb have "closed" and "opened".

Reply:
done